### PR TITLE
Create content/es/runtime.md

### PR DESCRIPTION
**Pull request by SullyMA**
### Describe your changes

I created a new file name runtime.md in Spanish in the content/es to match the English file in content/en

### Checklist before opening this PR (put `x` in the checkboxes)
- [X ] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [X ] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/es/runtime.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Los *container runtimes* son un componente necesario de los orquestadores de contenedores como Kubernetes. Manejan el ciclo de vida del contenedor, lo cual hace principalmente tres cosas. Primero, define cómo se especifican las imágenes de los contenedores y cómo el *runtime* puede recuperarlas. En segundo lugar, gestionan cómo estas imágenes se descomprimen, 
+se apilan, se montan y se ejecutan. Además, los *runtimes* gestionan los recursos de hardware, encargándose de todas estas acciones a nivel del sistema operativo. Esto incluye la asignación y la aislamiento de recursos.
+Con el tiempo, diferentes productos de *container runtime* evolucionaron, llevando a la Especificación OCI, que se convirtió en el estándar para los *container runtimes*.
+
+Introducir este estándar permite a los usuarios finales combinar cualquier *runtime* compatible con OCI con cualquier orquestador de contenedores compatible con OCI (como Kubernetes).
+
+## Related terms
+
+- [Cloud Native](https://glossary.cncf.io/cloud-native-apps/)
```
Comment (ramrodo) [inline content/es/runtime.md:26]:
```suggestion
- [Cloud Native](https://glossary.cncf.io/es/cloud-native-apps/)
```

File: content/es/runtime.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+- [Containerization](https://glossary.cncf.io/containerization/)
```
Comment (ramrodo) [inline content/es/runtime.md:27]:
```suggestion
- [Contenerización](https://glossary.cncf.io/es/containerization/)
```

File: content/es/runtime.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+- [Container Orchestration](https://glossary.cncf.io/container-orchestration/)
```
Comment (ramrodo) [inline content/es/runtime.md:28]:
```suggestion
- [Orquestación de Contenedores](https://glossary.cncf.io/es/container-orchestration/)
```

File: content/es/runtime.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+- [Microservices Architecture](https://glossary.cncf.io/microservices-architecture/)
```
Comment (ramrodo) [inline content/es/runtime.md:29]:
```suggestion
- [Arquitectura de Microservicios](https://glossary.cncf.io/es/microservices-architecture/)
```

File: content/es/runtime.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+---
+title: Runtime
+status: Completed
+category: concept
+tags: ["application", "", ""]
+---
+
+Un *runtime*, en general, ejecuta un fragmento de software. Es una [abstracción](https://glossary.cncf.io/abstraction/) del sistema operativo subyacente que traduce los comandos del programa en acciones correspondientes para el sistema operativo.
```
Comment (ramrodo) [inline content/es/runtime.md:8]:
```suggestion
Un *runtime*, en general, ejecuta un fragmento de software. Es una [abstracción](/es/abstraction/) del sistema operativo subyacente que traduce los comandos del programa en acciones correspondientes para el sistema operativo.
```

File: content/es/runtime.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+En el contexto de [cloud native](https://glossary.cncf.io/runtime/), *runtime* generalmente se refiere al *container runtime*. Un *container runtime* implementa específicamente la especificación de la [Open Container Initiative](https://opencontainers.org/) para asegurar un manejo consistente a través de diferentes tecnologías de orquestación de contenedores.
```
Comment (ramrodo) [inline content/es/runtime.md:10]:
```suggestion
En el contexto de [cloud native](/es/cloud-native-apps/), *runtime* generalmente se refiere al *container runtime*. Un *container runtime* implementa específicamente la especificación de la [Open Container Initiative](https://opencontainers.org/) para asegurar un manejo consistente a través de diferentes tecnologías de orquestación de contenedores.
```

File: content/es/runtime.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+**Problema que resuelve**
```
Comment (ramrodo) [inline content/es/runtime.md:12]:
```suggestion
## Problema que aborda
```

File: content/es/runtime.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+Sin la abstracción de un *container runtime*, la aplicación tendría que lidiar con toda la mecánica de cada sistema operativo, aumentando la complejidad de ejecutar la aplicación.
+
+**Cómo ayuda**
```
Comment (ramrodo) [inline content/es/runtime.md:16]:
```suggestion
## ¿Cómo ayuda?
```

**Review by ramrodo — CHANGES_REQUESTED** [review]
Thanks, Sully!

I left some suggestions.

File: content/es/runtime.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+Los *container runtimes* son un componente necesario de los orquestadores de contenedores como Kubernetes. Manejan el ciclo de vida del contenedor, lo cual hace principalmente tres cosas. Primero, define cómo se especifican las imágenes de los contenedores y cómo el *runtime* puede recuperarlas. En segundo lugar, gestionan cómo estas imágenes se descomprimen, 
+se apilan, se montan y se ejecutan. Además, los *runtimes* gestionan los recursos de hardware, encargándose de todas estas acciones a nivel del sistema operativo. Esto incluye la asignación y la aislamiento de recursos.
+Con el tiempo, diferentes productos de *container runtime* evolucionaron, llevando a la Especificación OCI, que se convirtió en el estándar para los *container runtimes*.
+
+Introducir este estándar permite a los usuarios finales combinar cualquier *runtime* compatible con OCI con cualquier orquestador de contenedores compatible con OCI (como Kubernetes).
+
+## Related terms
```
Comment (ramrodo) [inline content/es/runtime.md:24]:
```suggestion
## Términos relacionados
```

**Comment by ramrodo** [comment]
@SullyMA also, we need to add the following word to the `content/es/.wordlist.txt` file (to have the test in green):

- runtimes

**Review by ramrodo — APPROVED** [review]
LGTM. Thanks!

File: content/es/runtime.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+---
+title: Runtime
+status: Completed
+category: concept
+tags: ["application", "", ""]
+---
+
+Un *runtime*, en general, ejecuta un fragmento de software. Es una [abstracción](/es/abstraction/) del sistema operativo subyacente que traduce los comandos del programa en acciones correspondientes para el sistema operativo.
```
Comment (electrocucaracha) [inline content/es/runtime.md:8]:
```suggestion
Un *runtime*, en general, ejecuta un fragmento de software.
Es una [abstracción](/es/abstraction/) del sistema operativo subyacente que traduce los comandos del programa en acciones correspondientes para el sistema operativo.
```

File: content/es/runtime.md [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+
+En el contexto de [cloud native](/es/cloud-native-apps/), *runtime* generalmente se refiere al *container runtime*. Un *container runtime* implementa específicamente la especificación de la [Open Container Initiative](https://opencontainers.org/) para asegurar un manejo consistente a través de diferentes tecnologías de orquestación de contenedores.
```
Comment (electrocucaracha) [inline content/es/runtime.md:10]:
```suggestion
En el contexto de [cloud native](/es/cloud-native-apps/), *runtime* generalmente se refiere al *container runtime*.
Un *container runtime* específicamente implementa la especificación de la [Open Container Initiative](https://opencontainers.org/) para asegurar un manejo consistente a través de diferentes tecnologías de orquestación de contenedores.
```

File: content/es/runtime.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+## Problema que aborda
+
+Sin la abstracción de un *container runtime*, la aplicación tendría que lidiar con toda la mecánica de cada sistema operativo, aumentando la complejidad de ejecutar la aplicación.
+
+## ¿Cómo ayuda?
+
+Los *container runtimes* son un componente necesario de los orquestadores de contenedores como Kubernetes. Manejan el ciclo de vida del contenedor, lo cual hace principalmente tres cosas. Primero, define cómo se especifican las imágenes de los contenedores y cómo el *runtime* puede recuperarlas. En segundo lugar, gestionan cómo estas imágenes se descomprimen, 
+se apilan, se montan y se ejecutan. Además, los *runtimes* gestionan los recursos de hardware, encargándose de todas estas acciones a nivel del sistema operativo. Esto incluye la asignación y la aislamiento de recursos.
+Con el tiempo, diferentes productos de *container runtime* evolucionaron, llevando a la Especificación OCI, que se convirtió en el estándar para los *container runtimes*.
```
Comment (electrocucaracha) [inline content/es/runtime.md:20]:
```suggestion
Los *container runtimes* son un componente necesario de los orquestadores de contenedores como Kubernetes.
Manejan el ciclo de vida del contenedor, lo cual hace principalmente tres cosas.
Primero, define cómo se especifican las imágenes de los contenedores y cómo el *runtime* puede recuperarlas.
En segundo lugar, gestionan cómo estas imágenes se descomprimen, se apilan, se montan y se ejecutan.
Por último, los *runtimes* gestionan los recursos de hardware, encargándose de todas estas acciones a nivel del sistema operativo.
Esto incluye la asignación y el aislamiento de recursos.
Con el tiempo, diferentes productos de *container runtime* evolucionaron, llevando a la Especificación OCI,
que se convirtió en el estándar para los *container runtimes*.
```

File: content/es/runtime.md [hunk 13]
```diff
@@ -0,0 +1,29 @@
+---
+title: Runtime
+status: Completed
+category: concept
```
Comment (electrocucaracha) [inline content/es/runtime.md:4]:
```suggestion
category: Concepto
```

File: content/es/runtime.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+tags: ["application", "", ""]
```
Comment (electrocucaracha) [inline content/es/runtime.md:5]:
```suggestion
tags: ["aplicación", "", ""]
```

**Review by electrocucaracha — CHANGES_REQUESTED** [review]

**Review by electrocucaracha — APPROVED** [review]

**Pull request merged**